src/App.js

```diff
@@ -139,10 +138,6 @@ class App extends React.Component {
                 result.substring(result.length - 1 + 1);
             }
           }
-        } else if ((result || "").includes(".")) {
```

Comment:
This condition is already covered by the code on line 48 to 51 in the onClick function.